ts-obs/cmd/timescaledbGetPassword.go

```diff
@@ -11,34 +10,43 @@ import (
 var timescaledbGetPasswordCmd = &cobra.Command{
 	Use:   "get-password",
 	Short: "Gets the TimescaleDB/PostgreSQL password for a specific user",
+	Args:  cobra.ExactArgs(0),
 	RunE:  timescaledbGetPassword,
 }
 
 func init() {
 	timescaledbCmd.AddCommand(timescaledbGetPasswordCmd)
-    timescaledbGetPasswordCmd.Flags().StringP("user", "u", "postgres", "user whose password to get")
+	timescaledbGetPasswordCmd.Flags().StringP("user", "U", "postgres", "user whose password to get")
 }
 
 func timescaledbGetPassword(cmd *cobra.Command, args []string) error {
-    var err error
-
-    if len(args) != 0 {
-        return errors.New("\"ts-obs timescaledb get-password\" requires 0 arguments")
-    }
-
-    var user string
-    user, err = cmd.Flags().GetString("user")
-    if err != nil {
-        return err
-    }
-
-    secret, err := kubeGetSecret("ts-obs-timescaledb-passwords")
-    if err != nil {
-        return err
-    }
-
-    pass := secret.Data[user]
-    fmt.Printf("Password: %v\n", string(pass))
-
-    return nil
+	var err error
+
+	var user string
+	user, err = cmd.Flags().GetString("user")
+	if err != nil {
+		return err
+	}
+
+	var name string
+	name, err = cmd.Flags().GetString("name")
+	if err != nil {
+		return err
+	}
+
+	var namespace string
+	namespace, err = cmd.Flags().GetString("namespace")
+	if err != nil {
+		return err
+	}
+
+	secret, err := KubeGetSecret(namespace, name+"-timescaledb-passwords")
+	if err != nil {
+		return err
+	}
+
+	pass := secret.Data[user]
+	fmt.Printf("Password: %v\n", string(pass))
```

Comment:
The grafana get pass cmd just prints out the password, we should unify the prints (either "Password: xxxx" or just "xxxx") but I'm not sure which is preferred by users :\

Reply:
I think it makes more sense to just print the password, it makes it easier to copy paste